internal/fs/dir_handle.go

```diff
@@ -242,21 +277,31 @@ func (dh *dirHandle) ensureEntries(ctx context.Context) (err error) {
 //
 // LOCKS_REQUIRED(dh.Mu)
 // LOCKS_EXCLUDED(du.in)
-func (dh *dirHandle) ReadDir(
+func (dh *DirHandle) ReadDir(
```

Comment:
Please add unit tests for readDir method also.

Reply:
Added unit tests for readdir with zero offset,non zero offset and insufficient buffer size.